### ISSUE TITLE
Design review workflow

### ISSUE THREAD

**Issue by glitchassassin** [glitchassassin/william-perkins#4]
Transcription issues should have a standard template, linking to the source PDF, and repeating contribution guidelines for transcriptions.

Once transcription is complete and approved, a new PR (or perhaps a new issue) should be started for creating a modern version of the transcription.

**Comment by glitchassassin** [comment]
Automation can perhaps be a future item.
